init

```diff
@@ -173,8 +173,13 @@ function getParams()
 function setWritable($root, $paths)
 {
     foreach ($paths as $writable) {
-        echo "      chmod 0777 $writable\n";
-        @chmod("$root/$writable", 0777);
+        if(is_dir("$writable")) {
```

Comment:
if current user is not able to write the dir it can not chmod it. the chmod is only to allow writing for other users e.g. webserver user.